test/boostTest.cpp

```diff
@@ -156,14 +155,27 @@ test_suite* init_unit_test_suite( int /*argc*/, char* /*argv*/[] )
 
 	initializeOptions();
 
-	bool disableSemantics = !solidity::test::EVMHost::getVM(solidity::test::CommonOptions::get().evmonePath.string());
-	if (disableSemantics)
+	bool disableSemantics = true;
+	try
 	{
-		cout << "Unable to find " << solidity::test::evmoneFilename << ". Please provide the path using -- --evmonepath <path>." << endl;
-		cout << "You can download it at" << endl;
-		cout << solidity::test::evmoneDownloadLink << endl;
+		disableSemantics = !solidity::test::EVMHost::checkVmPaths(solidity::test::CommonOptions::get().vmPaths);
+	}
+	catch (std::exception const& _exception)
+	{
+		cerr << "Error: " << _exception.what() << endl;
+		exit(1);
+	}
+	if (disableSemantics)
 		cout << endl << "--- SKIPPING ALL SEMANTICS TESTS ---" << endl << endl;
+
+	cout << "Loaded EVMC VM's:" << endl;
+	for (auto &vmPath : solidity::test::CommonOptions::get().vmPaths)
+	{
+		evmc::VM& vm = solidity::test::EVMHost::getVM(vmPath.string());
+		cout << "- " << vm.name() << " v" << vm.version()  << " [caps=0x" << hex << vm.get_capabilities() << dec << ", path=" << vmPath.string() << "]" << endl;
```

Comment:
Is this hex code readable?

Reply:
Yes, it's either `0x1` for an evm1 vm or `0x2` for ewasm.

Reply:
I would say this should not belong to the regular test output. `isoltest` should only print the names of the tests it runs or an error if something goes wrong.

Reply:
Ok, I will remove that.